### PR TITLE
quick takes and popular comments on the lw home page

### PR DESCRIPTION
**Pull request by b0b3rt**
We adopt, with some stylistic changes, the Popular Comments section, as well as the Quick Takes section (including renaming Shortform to Quick Takes where necessary).

The main changes to Popular Comments are:
- single line metainfo (with the post title wrapping to the next line on mobile)
- comments are still expandable by clicking on them, but no longer collapsible that way (or at all); we also remove the cursor pointer when comments are expanded.
- adds a partial index for `getPopularComments` query, which is otherwise very slow on a cold db cache

The main changes to Quick Takes are:
- adding an "Cancel" button which collapses the entry form back to its initial state (and removes any content in the editor at the time)
- metainfo row in a quick takes list item no longer changes in styling/arrangement between collapsed/expanded states
- we hide the comment count element on mobile portrait displays, since we have much less space with our voting elements
- (not a change to quick takes per se) we remove the existing "New Shortform" button from the Recent Discussion section title

**Popular Comments**
Desktop view, with one of the comments expanded:
image

Mobile view:
image

**Quick Takes**
Desktop view, with one of the comments expanded
image

Mobile view
image

New Quick Take dialog (from user menu)
image

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1201302964208280/1206596867854136) by [Unito](https://www.unito.io)

### CODE REVIEW

**Comment by Discordius** [comment]
I think we should truncate the post title instead of having this happen: 
image

**Comment by Discordius** [comment]
My guess is we want to activate the wrapping behavior at a smaller breakpoint

**Comment by Discordius** [comment]
My vague vibe is also that the top comments section is a bit too long-lived. I feel like I am a bit more interested in seeing the top comments of the last 24-48 hours, but I am getting a lot of 3-5 day old comments

**Review by oetherington — COMMENTED** [review]
No objections from the EA Forum for the shared code (I've not tested the LW specific stuff).

There's also a field in the posts admin options called `hideFromPopularComments` that you might want to un-hide on LW for moderation purposes (it just excludes all comments on that post from being in the popular comments section).

Also, I see a commit message that says it should increase the recency bias (which I think we're planning to do on the EA forum soon too) but I don't see the code to do that - maybe you forgot to commit it?

**Comment by b0b3rt** [comment]
> Also, I see a commit message that says it should increase the recency bias (which I think we're planning to do on the EA forum soon too) but I don't see the code to do that - maybe you forgot to commit it?

Huh, I think I lost the change I made in `commentResolvers` somehow.  Good catch!

File: packages/lesswrong/components/alignment-forum/AlignmentForumHome.tsx [hunk 1]
```diff
@@ -47,6 +47,8 @@ const AlignmentForumHome = ({classes}: {
         </SectionTitle>
         <PostsList2 terms={recentPostsTerms} />
       </SingleColumnSection>
+      <QuickTakesSection />
+      <EAPopularCommentsSection />
```
Comment (Discordius) [inline packages/lesswrong/components/alignment-forum/AlignmentForumHome.tsx:51]: Should this be renamed? A bit confusing given that it's no longer EA-Forum only

File: packages/lesswrong/components/users/UsersMenu.tsx [hunk 2]
```diff
@@ -311,7 +311,8 @@ const UsersMenu = ({classes}: {
               />
               {currentUser.shortformFeedId &&
                 <DropdownItem
-                  title={isFriendlyUI ? "Your quick takes" : "Shortform Page"}
+                  // TODO: get Habryka's take on what the title here should be
```
Comment (Discordius) [inline packages/lesswrong/components/users/UsersMenu.tsx:314]: TODO can be removed

**Comment by Discordius** [comment]
The "parent-scroll" thing is still broken: 

image

**Comment by Discordius** [comment]
I wonder, is there a way in which we can avoid the comment loading spinner when expanding shortform comments? I find it really distracting, and I am a bit surprised it's worth saving the data.

**Comment by Discordius** [comment]
Oh, hmm, I do think that if the comment is a child of another comment, it is really important to somehow at least indicate that, and ideally add the "expand parent" button that we have in other contexts. Sparked by this comment: 

image

**Comment by Discordius** [comment]
Ok, I think that's all comments I have

**Review by Discordius — APPROVED** [review]

**Pull request merged**